docs/source/install.md

```diff
@@ -87,8 +87,18 @@ pytest pyogrio/tests
 
 ### Windows
 
-Install GDAL from an appropriate provider of Windows binaries. We've heard that
-the [OSGeo4W](https://trac.osgeo.org/osgeo4w/) works.
+[vcpkg](https://vcpkg.io/en/index.html) is used to build pyogrio from source
+as part of creating the Pyogrio Python wheels for Windows. You can install
+GDAL and other dependencies using vcpkg, and then build Pyogrio from source.
+
+See `.github/workflows/release.yml` for details about how vcpkg is used as part
+of the wheel-building process.
+
+We do not yet have instructions on building Pyogrio from source using vcpkg for
+local development; please feel free to contribute additional documentation!
+
+Alternatively, you can install GDAL from an appropriate provider of Windows
+binaries. We've heard that the [OSGeo4W](https://trac.osgeo.org/osgeo4w/) works.
```

Comment:
This was like this before as well, but my first go-to for having GDAL on windows (and build pyogrio from source) would be to use conda .. We mention conda above in the general installation section, but should we mention that here again for being explicit? When you just read the "Development" section for Windows, it might sound you need either vcpkg or something like OSGeo4W, while conda is still the easiest solution I would say.

Reply:
Good suggestion.  I assume you still need to set the environment variables when using conda, right?

Reply:
Yes, with our current setuptools-based building, using an environment variable to point to GDAL is still needed on Windows